src/query/api/v1/handler/prometheus/handleroptions/options.go

```diff
@@ -30,6 +32,7 @@ type PromWriteHandlerForwardingOptions struct {
 	// MaxConcurrency is the max parallel forwarding and if zero will be unlimited.
 	MaxConcurrency int                                    `yaml:"maxConcurrency"`
 	Timeout        time.Duration                          `yaml:"timeout"`
+	Retry          retry.Configuration                    `yaml:"retry"`
```

Comment:
Hey, so I think it'd be best to add some defaults here and make this a pointer type?

i.e. check if nil, if so use defaults, otherwise use the provided config.

Default config I think we want is:
```go
initialBackoff: 1s
backoffFactor: 2
maxRetries: 1
forever: false
jitter: true
```